src/ray/protobuf/ray_client.proto

```diff
@@ -139,6 +139,7 @@ message ClusterInfoType {
     CLUSTER_RESOURCES = 2;
     AVAILABLE_RESOURCES = 3;
     RUNTIME_CONTEXT = 4;
+    IS_ALIVE = 5;
```

Comment:
Not sure if a protocol change is strictly necessary -- we know it's alive, for instance, if the handshake from the data channel completes (which is what you'd want to test anyway)

Reply:
Yeah agreed; I'm ok with this though as an explicit check since it seems straightforward.